redi-connect-front/src/pages/front/signup/SignUpComplete.tsx

```diff
@@ -1,44 +1,62 @@
 import React from 'react'
-import { LoggedOutLayout } from '../../../layouts/LoggedOutLayout'
-import { RouteComponentProps } from 'react-router'
+import AccountOperation from '../../../components/templates/AccountOperation'
+import { Columns, Heading } from 'react-bulma-components'
+import Teaser from '../../../components/molecules/Teaser'
+import { useParams } from 'react-router'
 
 interface RouteParams {
   type: string
 }
 
-export default function SignUpComplete ({
-  match: {
-    params: { type }
-  }
-}: RouteComponentProps<RouteParams>) {
-  // TODO: Replace 'any' with whatever is TS-appropriate
-  // const Comp: any = (props: any) => <Link {...props} to="/app/dashboard" />;
+export default function SignUpComplete () {
+  const { type } = useParams<RouteParams>()
+
   return (
-    <LoggedOutLayout>
-      {type === 'public-sign-up-mentee-pending-review' && (
-        <>
-          <p>
-            Thanks for registering! We're thrilled that you're ReDI :) But there
-            is one last step before we can activate your profile because we want
-            to make sure you find the right mentor:
-          </p>
-          <p>
-            Simply contact Isabelle via e-mail isabelle@redi-school.org to
-            schedule a meeting. Or just turn up anytime between 5 and 8 on a
-            Tuesday at ReDI School.
-          </p>
-          <p>You can also find and write to Isabelle on the ReDI Slack.</p>
-        </>
-      )}
-      {type === 'public-sign-up-mentor-pending-review' && (
-        <>
-          <p>
-            Thanks for registering! We're thrilled that you're ReDI :) We
-            promise to review your profile as quickly as possible. We'll send
-            you an email once we're done.
-          </p>
-        </>
-      )}
-    </LoggedOutLayout>
+    <AccountOperation>
+      <Columns vCentered>
+        <Columns.Column
+          size={5}
+          responsive={{ mobile: { hide: { value: true } } }}
+        >
+          <Teaser.Isabelle />
+        </Columns.Column>
+
+        <Columns.Column size={5} offset={2}>
+          <Heading
+            size={1}
+            weight="normal"
+            renderAs="h1"
+            responsive={{ mobile: { textSize: { value: 2 } } }}
+            className="title--border"
+            spaced={true}
+          >
+            Welcome to ReDI Connect
+          </Heading>
+
+          <Heading size={4} renderAs="p" subtitle>
+            {type === 'mentee' && (
+              <>
+                Thanks for registering! We're thrilled that you're ReDI :) But there
+                is one last step before we can activate your profile because we want
+                to make sure you find the right mentor:
+                <br /><br />
```

Comment:
This is a nice shortcut but not the "semantic html way". Can we use three `<p>`-s instead?

Reply:
Sure, it's super ugly, but thank you for not being mean here. I leaf it like this because the text is not final and also some of the states need to be cleared. 
So if its fine for you let's keep it for now like this.